### ISSUE TITLE
Couple of issues

### ISSUE THREAD

**Issue by yellowcooln** [esmith443/Verizon-ONT-Bypass#1]
It looks like you updated your stick and didn't mention it in the setup.

I was able to update mine to `240408` from `220923`. If you don't update the firmware, **you are not able to save a blank PLOAM Password.** 

I was able to upgrade the firmware to `240408` using this GitHub issue. [https://github.com/Anime4000/RTL960x/issues/291](https://github.com/Anime4000/RTL960x/issues/291) Then I was able to save a blank PLOAM Password. 

Once installing in my UDM I was able to connect my fiber line in, but when i run the command for the IP tables. I get the following error:

```
Error: either "local" is duplicate, or "iptables" is a garbage.
```
I've tried changing the `192.168.1.0/24` to `192.168.0.0/24` to fit my default network but still the same error. 

I had to fall back to my ONT, as I didn't want to keep disturbing the family on a Thursday night. I will be able to try again Saturday morning as most of the family will be gone for the day. 

I also tried setting the Static IP on the WAN to `192.168.1.10` but was still not able to access the ONT webui. 

Any insight would be appreciated if you have any.

**Comment by yellowcooln** [comment]
Hello, I am just going to document going forward what I've tested with my configuration. I am not using the same model as you are. So this might cause some issues. G-211M-C.

I figured out my issue on this side of thing.

I needed to have a line break in the commands.

```
ip addr add dev eth9 local 192.168.1.100/24
iptables -t nat -A POSTROUTING -o eth9 -d 192.168.1.0/24 -j SNAT --to 192.168.100
```

I was able to access the WebUI after that but not able to get a VLAN tag, but I was able to get a GPON Status of `O5`. So because of that I had to fall back to my ONT, which worked no issues. 

Tomorrow I am going to try with my actually desktop or spare server and test it out again. 

I am also in the 8311 discord if anyone wants to chat.

**Comment by yellowcooln** [comment]
After doing some more testing this morning, I moved my Mellanox ConnectX-2 NIC out of my desktop and into my test bench and was able to connect to the ONT WebUI.

In the WebUI again I confirmed I'm getting a success on my registered status's and i have `05` status

![image](https://github.com/user-attachments/assets/715e7615-45f6-4767-a310-7475685e5172)

But once i go to the `VLAN Settings` Page I see no VLAN tag from Verizon like you did. 
![image](https://github.com/user-attachments/assets/a5d03396-0527-4194-9224-df78db7600db)

I tried setting my `Product Class` to `G-211M-C` to see if that would help but didn't.

**Comment by esmith443** [comment]
Come over to the 8311 discord server to discuss your issues https://discord.gg/8311

**Comment by yellowcooln** [comment]
I have returned my ONU and I'm just going to stick with my RJ45 to SFP+. I am currently happy with my ping now that I'm off cable, (I averaged about 40-45ms). So digging more down this path is just not realistic for me. 
![image](https://github.com/user-attachments/assets/a53b598c-7fcd-42af-93e8-7fe01ee1cc68)
It would be bad ass if it worked though. 

Thank you for helping me with this on Discord.
